### PR TITLE
Feature/df 590 e2e

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html [hunk 1]
```diff
@@ -41,9 +42,9 @@ <h2 class="sr-only">
       <alv-filter-panel (filterPanelValueChange)="onFiltersChange($event)"
                         [filterPanelValues]="jobSearchFilter$ | async">
       </alv-filter-panel>
-    </ng-container>
+    </div>
 
-    <ng-container class="main-column">
+    <div class="main-column">
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html]: @99999989 Does not have this any negative impact on the layout?

File: alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html [hunk 2]
```diff
@@ -61,9 +62,9 @@ <h2 class="sr-only">
           {{'global.common-phrases.no-results-found' | translate}}
         </div>
       </ng-container>
-    </ng-container>
+    </div>
 
-    <ng-container class="right-column">
+    <div class="right-column">
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search/job-search/job-search.component.html]: @99999989 Does not have this any negative impact on the layout?

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/job-search.po.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  searchResultsPanel = new SearchResultsPanelPo();
+  mainFilterPanel = new MainFilterPanelPo();
+  filterPanel = new FilterPanelPo();
+
+
+  navigateTo() {
+    return browser.get('/job-search');
+  }
+
+  scrollToBottom(): wdpromise.Promise<void>  {
```
Comment (99999989) [inline alv-portal-ui/e2e/src/job-advertisement/job-ad-search/job-search.po.ts]: Could be moved to somewhere where it's available for all other POs. In a separate utils file maybe.

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      .then(() => browser.sleep(SLEEP_TIME));
+  }
+
+  setContractType(value: ContractTypes) {
+    const contractTypeSelect = this.sectionElementFinder.$(alvFormControlName('contractType'));
+    return contractTypeSelect.click()
+      .then(() => this.clickOnOption(value, contractTypeSelect));
+  }
+
+  // todo just sends keys to the input, doesn't clear it first!
```
Comment (99999989) [inline alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts]: Is the comment still valid?

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/search-results-panel.po.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  get resultCount(): wdpromise.Promise<number> {
+    return this.results.then((results: ElementFinder[]) => results.length);
+  }
+
+  get resultHeaders(): wdpromise.Promise<string[]> {
+    return this.results.then((results: ElementFinder[]) => Promise.all(results.map(res => res.getText())));
+  }
+  //todo not checked yet
```
Comment (99999989) [inline alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/search-results-panel.po.ts]: Is this comment still valid?

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import { $, browser, promise as wdpromise } from 'protractor';
+import { alvFormControlName, getByTest } from '../../job-publication/selector-utils';
+
+export enum ContractTypes {
+  ALL = '0: ALL',
+  TEMPORARY = '1: TEMPORARY',
+  PERMANENT = '2: PERMANENT'
+}
+
+export const SLEEP_TIME = 2000;
```
Comment (99999989) [inline alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts]: Could be moved to somewhere where it's available for all other POs. In a separate utils file maybe. Or else do not export it.

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts (hunk 4 above)
Comment (ganqqwerty) [reply alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts]: yes, it's valid. I will need to do a better job at putting text to the input: clearing it somehow first and then sending the text

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/search-results-panel.po.ts (hunk 5 above)
Comment (ganqqwerty) [reply alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/search-results-panel.po.ts]: yes

File: alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts (hunk 6 above)
Comment (ganqqwerty) [reply alv-portal-ui/e2e/src/job-advertisement/job-ad-search/sections/filter-panel.po.ts]: I agree

**Pull request merged**